.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -146,7 +146,10 @@ def get_input(question, default=""):
         loaded_defaults = json.load(conf_file)
     #no need to authenticate on a worker machine (no website)
     if not args.worker:
-        loaded_defaults['authentication_method'] = 1 if loaded_defaults['authentication_method'] == 'PamAuthentication' else 2
+        if 'authentication_method' in loaded_defaults:
+            loaded_defaults['authentication_method'] = 1 if loaded_defaults['authentication_method'] == 'PamAuthentication' else 2
+        else:
+            loaded_defaults['authentication_method'] = 2
```

Comment:
The else branch shouldn't exist. The "default" value of the fields is handled by the "defaults" dictionary above the outer outer if block and this is just going to be confusing.